src/components/application_manager/test/commands/mobile/put_file_test.cc

```diff
@@ -234,6 +234,7 @@ TEST_F(PutFileRequestTest, Run_BinaryDataGreaterThanAvaliableSpace_UNSUCCESS) {
 
   const uint32_t avaliable_space = 0u;
   EXPECT_CALL(*mock_app_, GetAvailableDiskSpace())
+      .WillOnce(Return(avaliable_space))
```

Comment:
@LitvinenkoIra Can we use Times(n) ? Everywhere

Reply:
@AByzhynar Done 006f1d4978b7437c3146b44cb080ec716a0bdaab